test/integration/graql/query/GraqlDeleteIT.java

```diff
@@ -110,527 +110,8 @@ public static void closeSession() {
-        exception.expect(GraqlSemanticException.class);
-        exception.expectMessage(GraqlSemanticException.deleteSchemaConcept(newType).getMessage());
-        tx.execute(Graql.match(x.type("new-type")).delete(x.isa("thing")));
-    }
-
     @Test(expected = Exception.class)
     public void whenDeleteIsPassedNull_Throw() {
         tx.execute(Graql.match(var()).delete((Statement) null));
     }
-
-
-    @Test
-    public void whenTypeDoesNotMatch_Throw() {
-        tx.execute(Graql.insert(var().isa("production")));
-
-        exception.expect(GraqlSemanticException.class);
-        exception.expectMessage("it is not of the required type (or subtype of)");
-        tx.execute(Graql.match(var("x").isa("production")).delete(var("x").isa("movie")));
-    }
-
-
-    @Test
-    public void whenDeletingAttributeOwnership_onlyOwnershipIsDeleted() {
-        List<ConceptMap> answers = tx.execute(Graql.parse("match $x isa thing, has attribute $a; get; limit 1;").asGet());
-        ConceptId ownerId = answers.get(0).get("x").id();
-        ConceptId attrId = answers.get(0).get("a").id();
-
-        tx.execute(Graql.parse("match $x id " + ownerId + "; $x has attribute $a; $a id " + attrId + "; delete $x has attribute $a;").asDelete());
-
-        assertExists(tx, var().id(ownerId.toString()));
-        assertExists(tx, var().id(attrId.toString()));
-        assertNotExists(tx, Graql.and(var().id(ownerId.toString()).has("attribute", "a"), var("a").id(attrId.toString())));
-    }
-
-    @Test
-    public void whenDeletingAttributeOwnershipWithSupertype_onlyOwnershipIsDeleted() {
-        // using `attribute`
-        List<ConceptMap> answers = tx.execute(Graql.parse("match $x isa thing, has title $a; get; limit 1;").asGet());
-        ConceptId ownerId = answers.get(0).get("x").id();
-        ConceptId attrId = answers.get(0).get("a").id();
-
-        tx.execute(Graql.parse("match $x id " + ownerId + "; $x has title $a; $a id " + attrId + "; delete $x has attribute $a;").asDelete());
-
-        assertExists(tx, var().id(ownerId.toString()));
-        assertExists(tx, var().id(attrId.toString()));
-        assertNotExists(tx, Graql.and(var().id(ownerId.toString()).has("title", "a"), var("a").id(attrId.toString())));
-
-        // using `thing` throws
-        answers = tx.execute(Graql.parse("match $x isa thing, has title $a; get; limit 1;").asGet());
-        ownerId = answers.get(0).get("x").id();
-        attrId = answers.get(0).get("a").id();
-
-        exception.expect(GraqlSemanticException.class);
-        exception.expectMessage("Cannot delete attribute ownership, concept [$a]");
-        exception.expectMessage("is not of required attribute type [thing]");
-        tx.execute(Graql.parse("match $x id " + ownerId + "; $x has title $a; $a id " + attrId + "; delete $x has thing $a;").asDelete());
-    }
-
-    @Test
-    public void whenDeletingAttributeOwnershipWithSubtype_Throw() {
```

Comment:
is this migrated? Just double check

Reply:
Yes and no - at the time that you wrote this comment, the scenario was in draft; I have since implemented it.

Furthermore, it should have been quick to verify that and it wasn't.

I'm updating my verification PR to group the delete.feature into subsections, named:
- Things
- Roleplayers
- Attribute ownerships
- Edge cases